examples/stage-2-shield-study/addon/bootstrap.js

```diff
@@ -126,6 +131,7 @@ async function chooseVariation() {
   return toSet;
 }
 
+// TODO marcrowo: Move to lib?
```

Comment:
I am open to better solutions.  

My goal also to have a bootstrap.js that doesn't require changes, because small diffs are hard to review.